server/views/partials/bottom_nav.ejs

```diff
@@ -0,0 +1,16 @@
+
+<!-- 
+Author: Multiverse Design
+Date: Nov-10-2021
+FileName : bottom_nav.js -->
+
+
```

Comment:
everything here could be placed on the footer.ejs as the footer is only containing some lines of bootstrap scripts for styling. not necessary to have two

Reply:
I have deleted the bottom_nav file